mkdocs/__main__.py

```diff
@@ -170,11 +172,12 @@ def cli():
 @click.option('--no-livereload', 'livereload', flag_value='no-livereload', help=no_reload_help)
 @click.option('--dirtyreload', 'livereload', flag_value='dirty', help=dirty_reload_help)
 @click.option('--watch-theme', help=watch_theme_help, is_flag=True)
+@click.option('-w', '--watch', help=watch_help, multiple=True, default=[])
```

Comment:
```suggestion
@click.option('-w', '--watch', help=watch_help, type=click.Path(), multiple=True, default=[])
```

Improves help message

```diff
-  -w, --watch PATH                A directory to watch for live reloading. Can
+  -w, --watch TEXT                A directory to watch for live reloading. Can
                                   be supplied multiple times.
```